docs/API/metadata.md

```diff
@@ -92,40 +92,50 @@ This defines the dataset resource, and contains metadata received from the
 Pbench Agent, including the full contents of a `metadata.log` file created
 while gathering results and during dataset packaging.
```

Comment:
Given that we're gearing up to accept files from Zathras et al., I wonder whether we want to waffle a bit here -- e.g., drop the direct reference to "Pbench Agent" and say something like "including the contents of a `metadata.log` file if included in the results".